app/src/main/res/layout/fragment_destination_library.xml

```diff
@@ -46,21 +47,23 @@
       app:layout_behavior="@string/appbar_scrolling_view_behavior">
 
       <androidx.recyclerview.widget.RecyclerView
-        tools:listitem="@layout/item_book"
         android:id="@+id/zimfilelist"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent"
+        tools:listitem="@layout/item_book" />
 
     </androidx.swiperefreshlayout.widget.SwipeRefreshLayout>
 
   </org.kiwix.kiwixmobile.core.utils.NestedCoordinatorLayout>
 
-
   <TextView
     android:id="@+id/file_management_no_files"
     style="@style/no_list_content_text"
     android:layout_gravity="center"
+    android:layout_marginBottom="@dimen/material_design_appbar_size"
     android:text="@string/no_files_here"
+    android:visibility="gone"
```

Comment:
we don't need this and it makes our preview in Android studio less useful

Reply:
tch, and it was already being specified in the style

Reply:
Should I remove it?

Reply:
Yes, it is redundant as the style already defines it